skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -79,7 +79,7 @@ inline BaseActorValues GetBaseActorValues(espm::Loader& espm, uint32_t baseId,
     } else {
       std::string errorMessage = fmt::format(
         "Unable to read RACE. formId: {}, raceId: {}", baseId, raceIdOverride);
-      throw std::runtime_error(errorMessage.c_str());
+      throw std::runtime_error(errorMessage);
```

Comment:
I see no sense in creating error message in a separate variable, but this is really minor. We can marge as is, I think